frontend/src/worksheets/FilterWorkSheets.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+
+function FilterWorkSheets(props) {
+  const { status, onStatusChange } = props
+
+  const handleInputChange = (event) => {
+    console.log('filter', event.target.value)
```

Comment:
A console.log-ot vegyük ki.

Reply:
Töröltem.

Reply:
De kimaradt az ALL állapot. Még ne menjen be.